### ISSUE TITLE
Accessibility Testing for BMT II, Payment History, Add alert box to page

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#96620]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@1copenut

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [ ] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

Color contrast testing was unremarkable, save for the yellow "warning" alert box. The yellow background and slightly darker border are not 3:1 contrast ratio to the white `#fff` page background.

I consider this item out of scope because this alert box is a design system component and in use across the site. If there's a ticket that should be filed with the VADS team, I'm happy to do so.

### Who completed the axe scans?

@1copenut

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

I started the view under test locally and verified there were zero (0) accessibility issues with the Chrome Axe browser plugin. I also verified there were adequate Axe checks in our Cypress e2e test ( `src/applications/disability-benefits/view-payments/tests/e2e/view-payments.cypress.spec.js` ).

### How did axe testing go?

No issues or concerns.

### Who completed the content zoom and reflow test?

@1copenut

### Content zoom and reflow checks

- [x] All page elements are readable and usable at 200% zoom
- [x] All page elements are readable and usable at 300% zoom
- [x] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

The new alert box was fully readable at 200, 300, and 400%. I tested locally with Chrome and Firefox.

### Who completed the keyboard navigation test?

@1copenut

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Keyboard testing was unremarkable. The new alert box has no focusable elements. The page was navigable using standard keyboard controls.

### Do you have any other results to share?

I tested the page with NVDA screen reader on Chrome and Firefox. Both screen readers recognized the heading level and read the alert box content properly.

I briefly tested the page with Windows Voice Access as well. The zoom, scroll, and page commands worked as expected.
